src/sentry/runner/commands/devservices.py

```diff
@@ -13,6 +13,29 @@
     os.environ["DOCKER_HOST"] = "unix://" + RAW_SOCKET_HACK_PATH
 
 
+def start_docker_client():
+    import subprocess
+    import sys
```

Comment:
I would move the imports to the module scope -- then you can annotate this function as `-> docker.Client` or whatever the return type is supposed to be (this file is apparently unchecked but we should probably get it typed at some point)